tests/gl_draw_elements.c

```diff
@@ -0,0 +1,53 @@
+#include <GL/gl.h>
+#include <GL/glut.h>
+
+typedef struct Vertex {
+    short           x, y, z;
+    unsigned char   c_r, c_g, c_b, c_a;
+    short           u,v;
+} Vertex;
+
+Vertex triangle[3] = {
+    0, 0, 0,   0, 255, 255, 255, 0, 0,
+    1, 1, 0, 255,   0, 255, 255, 0, 0,
+    1, 0, 0, 255, 255,   0, 255, 0, 0
```

Comment:
Err, nevermind the above, I was somehow reading this to be filling raw bytes, but it's filling elements, so all good.